r-package/lint_package.R

```diff
@@ -0,0 +1,15 @@
+Sys.setenv(LINTR_COMMENT_BOT='false')
```

Comment:
👍 I haven't found those bots to be useful in the past, and they can make PRs harder to read.